index.js

```diff
@@ -414,7 +414,7 @@ class LedgerBridgeKeyring extends EventEmitter {
     })
 
     if (success) {
-      let v = payload.v - 27
+      let v = parseInt(payload.v, 10)
```

Comment:
LGTM - I believe we also want to apply this fix for the `personal_sign` messages. Can we update the `signPersonalMessage` method above with the same change, or update the PR description to mention this is a partial or more specified fix? @lambertkevin

Reply:
Hi @digiwand ! Sry for the late answer, i was off for a few days. 
Yes you're right, I updated the `signPersonalMessage` as well, that should make it work for EIP-191 messages verification as well this way 👌

Reply:
no sorries needed! Hope you're taking care and making the most of life :)

Thanks for update! looking now